web/src/lib/components/admin-page/delete-confirm-dialoge.svelte

```diff
@@ -23,25 +23,16 @@
 	};
 </script>
 
-<div
-	class="border bg-immich-bg dark:bg-immich-dark-gray dark:border-immich-dark-gray p-4 shadow-sm w-[500px] max-w-[95vw] rounded-3xl py-8 dark:text-immich-dark-fg"
+<ConfirmDialogue
+	title="Delete User"
+	confirmText="Delete"
+	on:confirm={deleteUser}
+	on:cancel
 >
-	<div
-		class="flex flex-col place-items-center place-content-center gap-4 px-4 text-immich-primary dark:text-immich-dark-primary"
-	>
-		<h1 class="text-2xl text-immich-primary dark:text-immich-dark-primary font-medium">
-			Confirm User Deletion
-		</h1>
-	</div>
-	<div>
-		<p class="ml-4 text-md py-5 text-center">
-			{user.firstName}
-			{user.lastName} account and assets along will be marked to delete completely after 7 days. are
-			you sure you want to proceed ?
-		</p>
-
-		<div class="flex w-full px-4 gap-4 mt-8">
-			<Button fullwidth color="red" on:click={deleteUser}>Confirm</Button>
+	<svelte:fragment slot="prompt">
+		<div class="flex flex-col gap-4">
+			<p><b>{user.firstName} {user.lastName}</b>'s account and his assets will queued for a definitive deletion after 7 days.</p>
```

Comment:
```suggestion
			<p>The account and assets belonging to <b>{user.firstName} {user.lastName}</b> will be permanently deleted after 7 days.</p>
```

The use of `'s` could read strangely with some names, and "his" is not gender neutral.